homeassistant/components/mqtt/config_flow.py

```diff
@@ -327,14 +332,20 @@ def validate_user_input(
 
 
 @callback
-def data_schema_from_fields(data_schema_fields: dict[str, PlatformField]) -> vol.Schema:
+def data_schema_from_fields(
+    data_schema_fields: dict[str, PlatformField],
+    reconfig: bool = False,
```

Comment:
I'd suggest to not make this an optional parameter to avoid mistakes in the future.